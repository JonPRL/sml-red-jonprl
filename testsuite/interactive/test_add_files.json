{"cmd": "new_session", "name": "add_files"}
{"cmd": "add_files", "session_id": "add_files", "filenames": ["testsuite/tests/example.prl"]}
