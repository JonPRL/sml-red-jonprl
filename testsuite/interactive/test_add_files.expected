{"session_id": "add_files"}
{"msg": "Files have been added"}
